lib/ui/src/components/sidebar/treeview/utils.ts

```diff
@@ -66,13 +59,32 @@ export const keyEventToAction = ({
 
 export const createId = (id: string, prefix: string) => `${prefix}_${id}`;
 
+export function isRoot(item: Item): item is Root {
+  if (item as Root) {
+    return item.isRoot;
+  }
+  return false;
+}
+export function isGroup(item: Item): item is Group {
+  if (item as Group) {
+    return !item.isRoot && !item.isLeaf;
+  }
+  return false;
+}
+export function isStory(item: Item): item is Story {
+  if (item as Story) {
+    return item.isLeaf;
+  }
+  return false;
+}
+
```

Comment:
These functions are duplicated from `https://github.com/storybookjs/storybook/pull/9820/files#diff-04d3ed821e8d485937cf46951931c2b1R247` should they be exported from `@storybook/api`?

Reply:
I agree